lib/ecto/multi.ex

```diff
@@ -305,6 +305,10 @@ defmodule Ecto.Multi do
     end
   end
 
+  @doc """
+  Transforms the `Ecto.Multi` into a list of operations to be performed. Inspecting
```

Comment:
I think it was my mistake. It should be documented & public API, we even mention it in moduledocs.